src/expression.peg

```diff
@@ -21,8 +21,9 @@ expr = { term ~ (operation ~ term)* }
 term = _{ ans | number | "(" ~ expr ~ ")" }
 
 set_directive = @{ "set " ~ (word ~ " ")* ~ word }
+tor_directive = { term ~ (operation ~ term)* ~ "to " ~ word }
```

Comment:
I think this can also be written as:
```
tor_directive = { expr ~ "to " ~ radix }
radix = { "dec" | "oct" | "hex" | "bin" }
```
IMO, this is cleaner and expresses the intent better.

Reply:
Ok, it's more Pest's idiomatic (I've never used Pest before).